### PR TITLE
Pull Models to Development

### PR DESCRIPTION
**Pull request by CalyCherkaoui**
In this mini-project, we tried to create database Models with Active:records. The models created are:
1- User
2- Post
3-Comment

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+
+  validates :title, presence: true, length: { minimun: 5 }, too_short: 'Get serious!!!'
+  validates :body, presence: true, length: { minimun: 5 }, too_short: 'Come on!!! This is toooooo short'
```
Comment (wathigo) [inline app/models/post.rb]: I think you meant `minimum1` instead of `minimum`

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required.
Great work so far.
Let's fix an issue that is raised upon post creation.

**REQUIRED**
* A post can be created **(covered by automated tests)** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/creatingPost.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails))
Screenshot from 2020-08-11 09-05-57

**After making the changes, submit another request for review.**

**Happy Coding  :computer: 
Simon Wathigo.**

File: app/models/post.rb (hunk 1 above)
Comment (asiimwedismas) [reply app/models/post.rb]: Issues raised have been fixed

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great progress @team :clap: However, some changes need to be implemented in order to get approval for this milestone. Check :arrow_down: 

- **[REQUIRED]** Post can be created. When I try to create a post, it raises the following error. kindly fix it
image

### That's it :relieved: When finished, kindly submit a code re-review request :pray: 
Happy coding :v: 
**_Ahmed Mahfoudh_**

**Comment by asiimwedismas** [comment]
All good!

**Review by rOluochKe — APPROVED** [review]
Hello @CalyCherkaoui and @asiimwedismas 
### Well done, you got this right :clap: it has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**